mocks/mocks.go

```diff
@@ -312,6 +312,11 @@ func (sa *StorageAuthority) CountPendingAuthorizations(_ context.Context, _ int6
 	return 0, nil
 }
 
+// DeactivateAuthorization is a mock
```

Comment:
There's a mismatch here between what the comment says (`DeactivateAuthorization`) and the function name (`UpdateAuthz`).